operator/v1/types_dns.go

```diff
@@ -30,6 +30,48 @@ type DNS struct {
 
 // DNSSpec is the specification of the desired behavior of the DNS.
 type DNSSpec struct {
+	// servers is a list of DNS resolvers that provide name query delegation for one or
+	// more subdomains outside the scope of the cluster domain. Valid Servers are prepended
+	// to the ConfigMap used to configure CoreDNS. If Servers consists of more than one
```

Comment:
This is a lot of implementation detail; does the user of this API care about the configmap or CoreDNS's listening port and servers?

Reply:
ack, I'll remove implementation details.